cmd/network_info.go

```diff
@@ -29,7 +29,7 @@ Shows information about the host network and interfaces or only from a specific
 
 		if len(args) > 0 {
 			inet := args[0]
-			command = inet + "/info"
+			command = "interface/" + inet + "/info"
```

Comment:
```suggestion
			command = "interface/{inet}/info"
```

Reply:
that will not work, need a lot more code :)